js/src/3rdparty/etherscan/util/address.js

```diff
@@ -0,0 +1 @@
+../../../abi/util/address.js
```

Comment:
This is Github way of showing a Symlink. This is related to what was discussed above.